deploy/helm/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: swo-k8s-collector
-version: 4.2.0-alpha.11
+version: 4.2.0-alpha.12
```

Comment:
I wouldn't include this into 4.2.0, this should be 4.3.0 material. We wanted to close 4.2.0 as soon as we can

Reply:
changed to 4.3.0-alpha.1